wcfsetup/install/files/lib/system/form/builder/container/PrefixConditionFormFieldContainer.class.php

```diff
@@ -0,0 +1,211 @@
+<?php
+
+namespace wcf\system\form\builder\container;
+
+use wcf\data\IStorableObject;
+use wcf\system\form\builder\data\processor\CustomFormDataProcessor;
+use wcf\system\form\builder\field\IFormField;
+use wcf\system\form\builder\field\IImmutableFormField;
+use wcf\system\form\builder\field\ISelectionFormField;
+use wcf\system\form\builder\IFormDocument;
+
+/**
+ * @author Olaf Braun
+ * @copyright 2001-2025 WoltLab GmbH
+ * @license GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @since 6.3
+ */
+final class PrefixConditionFormFieldContainer extends FormContainer
+{
+    /**
+     * form field to which the prefix selection is added
+     */
+    protected IFormField $field;
+
+    /**
+     * selection form field containing the prefix options
+     */
+    protected ?ISelectionFormField $prefixField;
```

Comment:
This contain exists for the sole purpose of adding a selection field in front of another form field. Why is it nullable?

What not use a regular form field when there is no prefix and only use this field when there is one, greatly simplifying the logic? Is there any reason to why this is allowed to be optional?